test/dpul_collections/indexing_pipeline/integration/full_integration_test.exs

```diff
@@ -21,18 +21,29 @@ defmodule DpulCollections.IndexingPipeline.FiggyFullIntegrationTest do
     # stays in the transformation cache so the total number of records must take
     # this into account.
     deleted_resource_count = FiggyTestSupport.deletion_marker_count()
-    # Resources that are not public or not complete
-    non_indexable_resource_count = 4
-    total_records = ephemera_folder_count + deleted_resource_count - non_indexable_resource_count
+    total_records = ephemera_folder_count + deleted_resource_count
     # Empty resources are resources with no image file sets
     empty_resource_count = 1
 
+    if System.get_env("DEBUG_INTEGRATION") do
```

Comment:
I'm not sure if we should keep this, but this is an area where I'm constantly inserting the same IO.puts statements. In the long term, I think we need a better (or more centralized) way to calculate what the expected counts are.

Reply:
If it's workin' for ya, happy to keep it around. Maybe this is a place we should add telemetry?